graql/language/rule-validation.feature

```diff
@@ -688,7 +688,7 @@ Feature: Graql Rule Validation
 
 #CHECK CLAUSE VALIDATION
```

Comment:
I'm not sure these comments (lines 689, 707, 746) add much value. We should probably just delete them.

You can add an optional scenario description below a scenario name - just any string that doesn't start with `When`, `Then`, `And` ... or any of the other Cucumber keywords. But let's only do this when necessary to clarify the intended purpose of the scenario!

Examples of this can be found in `recursion.feature`.

Reply:
Alright, put these comments into the Scenario description.